### ISSUE TITLE
Port status: password_policy

### ISSUE THREAD

**Issue by bennybobw** [backdrop-ops/contrib#839]
**Name of the module, theme, or layout you are working on**
Password Policy 

**Link to the drupal.org module, theme, or layout**
[https://www.drupal.org/project/password_policy/](https://www.drupal.org/project/password_policy/)

**(Optional) Link to the drupal.org issue where you notified the Drupal maintainer(s) of this port**
https://www.drupal.org/project/password_policy/issues/3485929

**Do you need assistance with this port?**
No

**Comment by bennybobw** [comment]
You know I looked at that and assumed it was different because the README doesn't say anything about including things like the password expiration. We're porting this for a client and we'd need to see if they'd be ok with the modifications, but I'll look into the module.

**Comment by bennybobw** [comment]
*Update*: I looked at this and passphrase_policy module has a pretty limited set of features compared to the other module. It doesn't provide any configuration options, all the restrictions are baked in, so I'm going to go ahead with the port.

**Comment by yorkshire-pudding** [comment]
@bennybobw - it looks like this is a duplicate of #763 that @neanhunt started and @jenlampton indicated would pick up.  I note someone has created an empty repo in backdrop-contrib: https://github.com/backdrop-contrib/password_policy

Whoever is furthest along, it would be good to have something that others can test and help with to get it ready for release.

**Comment by bennybobw** [comment]
Yeah you know, our client actuallydecided to go with passphrase_policy instead so we wont' be porting it!

**Comment by jenlampton** [comment]
> I note someone has created an empty repo in backdrop-contrib: [backdrop-contrib/password_policy](https://github.com/backdrop-contrib/password_policy)

I have deleted this, my client also went with passphrase policy :)
